### PR TITLE
Use illustrations for appearance setting

### LINKED ISSUE #215: Color Scheme Chooser
We need to come up with an interface for the prefers-color-scheme setting.

We should bear in mind that this setting has 3 states: prefer `dark`, prefer `light`, and `no-preference` (maybe we can call this mixed or automatic or something more palatable). It seems like most major platforms only handle `no-preference` and `dark`.

I think we want to do a slow rollout and start with only shell elements, so the first version of this should probably be two illustrations depicting light shell elements and dark shell elements. By shell elements I mean the dock and panel popovers.

It might be worth depicting mixed windows so that the user knows this only changes shell elements and not apps.

Future versions of this selector should probably be three illustatrations:
* One with light shell elements and app windows
* One with light shell elements and mixed app windows
* One with dark shell elements and dark app windows

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/91296677-color-scheme-chooser?utm_campaign=plugin&utm_content=tracker%2F57462545&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F57462545&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by danrabbit**
Fixes #215

### CODE REVIEW

**Comment by cassidyjames** [comment]
@danrabbit I don't like "Appearance" repeated there, because you're changing the System Settings → Desktop → Appearance → Appearance setting, which seems weird. I think I had "Visual style" or "Style preference" or just "Style" in my sketches. I also wonder if we should keep the descriptive text down below to ensure it's still obvious what this setting does, and just keep "Default" or "Dark" in the tooltip?

File: src/Views/Appearance.vala [hunk 1]
```diff
@@ -38,18 +38,36 @@ public class Appearance : Gtk.Grid {
         row_spacing = 6;
         margin_start = margin_end = 6;
 
-        var dark_label = new Gtk.Label (_("Prefer dark style:"));
+        var dark_label = new Gtk.Label (_("Appearance:"));
```
Comment (cassidyjames) [inline src/Views/Appearance.vala:41]:
```suggestion
        var dark_label = new Gtk.Label (_("Style:"));
```

**Comment by cassidyjames** [comment]
Ah yeah here was a quick sketch from the other day. It has some issues, i.e. we don't do radios and text labels below the images and I didn't have room to put the "Style preference" label on the left. :stuck_out_tongue:  But sharing here for context, anyway.

IMG_20200423_195818

**Comment by danrabbit** [comment]
It doesn't seem to be possible to put the radios below the grid, but I can add labels

**Pull request merged**